# file: C:\Users\ADITI\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\scipy\stats\_binned_statistic.py
# hypothesis_version: 6.135.14

[0.5, 'Internal Shape Error', 'bin_edges', 'binned_statistic', 'binned_statistic_2d', 'binned_statistic_dd', 'binnumber', 'count', 'ignore', 'max', 'mean', 'median', 'min', 'statistic', 'std', 'sum', 'x_edge', 'y_edge']